x/lockup/keeper/lock_test.go

```diff
@@ -877,6 +1001,90 @@ func (suite *KeeperTestSuite) TestSlashTokensFromLockByID() {
 	suite.Require().Error(err)
 }
 
+func (suite *KeeperTestSuite) TestSlashTokensFromLockByIDSendUnderlyingAndBurn() {
```

Comment:
Also wondering why this isn't table-driven

Reply:
Table driven added here https://github.com/osmosis-labs/osmosis/pull/4884/commits/7754fd0f6b24c12bb3511fe7d7b16f5d3d9b0c41